compose/docker-compose.yml

```diff
@@ -19,12 +19,27 @@ services:
       - appdata:/var/www/html
       - sockdata:/sock
       - ssldata:/etc/nginx/certs
+    healthcheck:
+      test: 'curl --fail http://127.0.0.1:8000'
+      interval: 1m30s
```

Comment:
Why is the `interval` a different value from the `docker-compose.dev.yml` file?

Reply:
I think remove all interval, timeout, and start-period will be fine.
Using their default value may a better choice

The options that can appear before CMD are:

```
--interval=DURATION (default: 30s)
--timeout=DURATION (default: 30s)
--start-period=DURATION (default: 0s)
--retries=N (default: 3)
```

Reply:
Cool, definitely prefer the defaults 👍